### PR TITLE
[workflow] create actions to update the sysdig-operator repo upon merges …

### PR DESCRIPTION
**Pull request by lilx1ao**
…to charts

## Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ ] Chart Version bumped
- [ ] Variables are documented in the README.md
- [ ] Title of the PR starts with chart name (e.g. `[mychartname]`)
- [ ] PR only contains changes for one chart

### CODE REVIEW

**Comment by lilx1ao** [comment]
this code hasn't been tested yet.

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -37,4 +37,4 @@ jobs:
           git add -A "charts/**/README.md"
           git add -A "charts/**/values.yaml"
           git pull && git commit -m "Updating README.md and values files" && git push
-          popd
+          popd
```
Comment (yashgiri) [inline .github/workflows/release.yaml:40]: just a nit but is it needed to remove the new line at the EOF.

**Review by yashgiri — COMMENTED** [review]
LGTM

File: .github/workflows/release.yaml (hunk 1 above)
Comment (lilx1ao) [reply .github/workflows/release.yaml:40]: no particular reasons. I tried adding things below and then removed them later hence the deletion.

**Review by achandras — APPROVED** [review]

**Pull request merged**